src/hugr/views.rs

```diff
@@ -315,6 +319,15 @@ where
         }
     }
 
+    fn get_function_type(&self, node: Node) -> Option<&FunctionType> {
```

Comment:
I think for the purposes this PR is working towards, this should always return the function type for the root. For purposes other than getting the function type _of a hugr_, people can use `get_nodetype` as you have